base/bsupport/logcustomcounterstub_test.go

```diff
@@ -0,0 +1,43 @@
+package bsupport
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStubLogCustomCounter(t *testing.T) {
+	reg, lookup := NewStubLogCustomCounterRegistry()
+	a := reg.RegisterCustomCounter("a")
+	b := reg.RegisterCustomCounter("b")
+	c := reg.RegisterCustomCounter("c")
+	d := reg.RegisterCustomCounter("d")
+	b2 := reg.RegisterCustomCounter("b")
+
+	a(3)
+	b(5)
+	c(7)
+	d(11)
+	b2(13)
+
+	{
```

Comment:
What's the value of this scoping? Aren't blank lines between the assertions enough? 

For this kind of repetitive asserting, [table-driven tests](https://dave.cheney.net/2019/05/07/prefer-table-driven-tests) would also work really well I think.

Reply:
simplified and moved to base/btest together with stubs for input interfaces.

The previous version was using fixed size array to hold counters and it needed more tests which are now unnecessary.